src/scenes/DeliverablesRouter/QuestionsDeliverableView.tsx

```diff
@@ -133,26 +132,28 @@ export type Props = EditProps & {
 
 const QuestionsDeliverableView = (props: Props): JSX.Element => {
   const { ...viewProps }: ViewProps = props;
+  const deliverableId = viewProps.deliverable.id;
   const projectId = viewProps.deliverable.projectId;
 
   const dispatch = useAppDispatch();
   const { isMobile } = useDeviceInfo();
   const { activeLocale } = useLocalization();
-  const { currentParticipantProject } = useParticipantData();
   const { currentDeliverable: deliverable } = useDeliverableData();
   const { status: requestStatus } = useUpdateDeliverable();
 
-  // TODO: replace with actual question/answer data
-  const ppsSearchResults = useAppSelector(selectParticipantProjectSpeciesListRequest(projectId));
+  const variablesWithValues = useAppSelector((state) =>
+    selectDeliverableVariablesWithValues(state, deliverableId, projectId)
+  );
+  console.log({ variablesWithValues });
 
   const [showSubmitDialog, setShowSubmitDialog] = useState<boolean>(false);
 
-  const submitButtonIsDisabled = useMemo(() => {
-    return (
-      !ppsSearchResults?.data?.length ||
-      ppsSearchResults?.data?.every((species) => species.participantProjectSpecies.submissionStatus === 'Approved')
-    );
-  }, [ppsSearchResults]);
+  // const submitButtonIsDisabled = useMemo(() => {
```

Comment:
I just commented this out for now, we may also need to plumb in some other "deliverable related" values in order to correctly enable/disable the button.